resources/js/Pages/Articles/Show.tsx

```diff
@@ -1,8 +1,25 @@
+import { Heading } from "@/Components/Heading";
+import { Img } from "@/Components/Image";
 import { DefaultLayout } from "@/Layouts/DefaultLayout";
 
 const ArticlesShow = ({ article }: { article: App.Data.Articles.ArticleData }): JSX.Element => (
     <DefaultLayout>
-        <div className="mx-6 sm:mx-8 2xl:mx-0">{article.title}</div>
+        <div className="mx-8 -mt-6 pb-6 pt-8 sm:-mt-8 md:px-24 lg:mt-0 lg:overflow-hidden lg:rounded-xl lg:border lg:border-theme-secondary-300">
```

Comment:
can you double check the padding as it feels too close to the image compared with the design